guides/common/modules/proc_using-the-project-content-dashboard.adoc

```diff
@@ -15,13 +15,13 @@ The following table shows the descriptions of the possible configuration states.
 [cols="1,3,5", options="header"]
 |====
 | Icon | State | Description
-| image:1.png[] | *Hosts that had performed modifications without error* | Host that successfully performed modifications during the last reporting interval.
-| image:2.png[] | *Hosts in error state* | Hosts on which an error was detected during the last reporting interval.
-| image:3.png[] | *Good host reports in the last 35 minutes* | Hosts without error that did not perform any modifications in the last 35 minutes.
-| image:4.png[] | *Hosts that had pending changes* | Hosts on which some resources would be applied but Puppet was configured to run in the `noop` mode.
-| image:5.png[] | *Out of sync hosts* | Hosts that were not synchronized and the report was not received during the last reporting interval.
-| image:6.png[] | *Hosts with no reports* | Hosts for which no reports were collected during the last reporting interval.
-| image:7.png[] | *Hosts with alerts disabled* | Hosts which are not being monitored.
+| image::common/1.png[] | *Hosts that had performed modifications without error* | Host that successfully performed modifications during the last reporting interval.
```

Comment:
Unrelated to this PR, but that's a very generic filename.

Reply:
Yeah I don't want to go into this rabbithole now :)